### PR TITLE
announcements: Post HPC interest survey

### PR DESCRIPTION
**Pull request by matthew-mcallister**
A bit long I admit, but not as long as the survey. I plan to post this by the evening.

This is also the last chance to make comments on the survey itself, as I'm going to lock it when I post this announcement.

http://supernova.ocf.berkeley.edu:8108/announcements/2017-03-01/hpc-survey

### CODE REVIEW

File: ocfweb/announcements/templates/announcements/2017-03-01-hpc-survey.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+              OCF HPC interest form
+            </a></p>
+            <p>
+              To anyone who may consider hosting a current or future project on
+              our open computer infrastructure, we urge you to fill out this
+              survey as best as you can, using estimations if you do not have
+              all the hard data. We want to see the full breadth of projects on
+              campus that have unmet computing needs. Our first server is
+              planned to feature multiple high-end graphics cards, a many-core
+              CPU, and spacious quantities of RAM and storage space, but we
```
Comment (NickImpicciche) [inline ocfweb/announcements/templates/announcements/2017-03-01-hpc-survey.html]: I would remove the world space at the end of the sentence, sounds awkward

Comment (matthew-mcallister) [inline ocfweb/announcements/templates/announcements/2017-03-01-hpc-survey.html]: Oh, my bad, thanks.

File: ocfweb/announcements/templates/announcements/2017-03-01-hpc-survey.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+              will need as much input as we can get to make informed decisions
+              if we want to make high-performance computing a reality for as
+              many as possible.
+            </p>
+            <p>
+              To student groups or projects in particular, we would like to
```
Comment (NickImpicciche) [inline ocfweb/announcements/templates/announcements/2017-03-01-hpc-survey.html]: + research group, what does projects in particular mean?

Is this open to everyone? Only student or research groups? Should clarify as well if we want even those not in these groups to fill out.

Comment (matthew-mcallister) [inline ocfweb/announcements/templates/announcements/2017-03-01-hpc-survey.html]: I think we want as many people as possible to fill out the survey.

Comment (NickImpicciche) [inline ocfweb/announcements/templates/announcements/2017-03-01-hpc-survey.html]: Then should say that as explicitly as possible

Comment (matthew-mcallister) [inline ocfweb/announcements/templates/announcements/2017-03-01-hpc-survey.html]: We do. However, I don't think research groups will be interested in a sketchy trial run, so I want to call the attention of students in particular to this paragraph.

Comment (NickImpicciche) [inline ocfweb/announcements/templates/announcements/2017-03-01-hpc-survey.html]: Okay, I still don't get what students or projects in particular means. What kind of project, who is that referring to?

Comment (matthew-mcallister) [inline ocfweb/announcements/templates/announcements/2017-03-01-hpc-survey.html]: "Student projects", I guess. Maybe it should just read "To student groups in particular"?

**Pull request merged**